test/connectors/ImporterAddressConnectorSpec.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import base.SpecBase
+import connectors.httpParsers.ResponseHttpParser.HttpGetResult
+import mocks.MockHttp
+import models.TraderAddress
+import org.scalatest.matchers.must.Matchers.convertToAnyMustWrapper
+import play.api.http.Status
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import uk.gov.hmrc.http.HttpResponse
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class ImporterAddressConnectorSpec extends SpecBase with MockHttp {
+
+  val errorModel: HttpResponse = HttpResponse(Status.NOT_FOUND, "Error Message")
+
+  val traderAddress: TraderAddress = TraderAddress("first", "second", Some("third"), "fourth")
```

Comment:
TraderAddress("first", "second", Some("third"), "fourth") is used in multiple files, should be a global variable

Reply:
complete